src/pages/notification/index.tsx

```diff
@@ -0,0 +1,161 @@
+import { NotificationItem } from '../../service/notification/types';
+import './style.less';
+
+interface INotificationCenterProps {
+  onClick?: () => void;
+}
+
+const IconFromNotification = (notification: NotificationItem) => {
+  switch (notification.type) {
+    case 'remote':
+      return <SyncOutlined />;
+    case 'customerService':
+      return <TeamOutlined />;
+
+    default:
+      return <SyncOutlined />;
+  }
+};
+
+const NotificationCenter = ({ onClick }: INotificationCenterProps) => {
+  const { notifications, fetchNotifications, hasUnread, markAllAsRead } = useNotification();
+
+  const [t] = useTranslation();
+
+  const { show } = useIntercom();
+
+  const fetch = () => {
+    const providerURL =
+      process.env.NODE_ENV === 'development'
+        ? NOTIFICATION_DEV_CONFIG_FILE_URL
+        : NOTIFICATION_RELEASE_CONFIG_FILE_URL;
+    fetchNotifications(providerURL);
+  };
+
+  useEffect(() => {
+    fetch();
+  }, []);
+
+  useInterval(() => {
+    fetch();
+  }, 5 * 60_000);
+
+  const NotificationList = () => {
+    return (
+      <div style={{ width: '500px', padding: '10px' }}>
+        <div style={{ fontSize: '24px', fontWeight: 'bold', marginTop: '10px' }}>
+          {t('general.notification.title')}
+        </div>
+        <List
+          itemLayout="vertical"
+          size="large"
+          pagination={{
+            pageSize: 5,
+          }}
+          dataSource={notifications.slice().reverse()}
+          renderItem={item => (
+            <List.Item key={item.id} style={{ padding: '16px 10px' }}>
+              <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+                <div
+                  style={{
+                    display: 'flex',
+                    alignSelf: 'start',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                  }}
+                >
+                  <div style={{ position: 'absolute', left: '-10px' }}>
+                    {!item.isRead && (
+                      <div
+                        style={{
+                          width: '8px',
+                          height: '8px',
+                          borderRadius: '4px',
+                          backgroundColor: '#1199FA',
+                          marginRight: '10px',
+                        }}
+                      />
+                    )}
+                  </div>
+                  <div style={{ width: '48px', height: '48px' }}>
+                    <Avatar
+                      size={48}
+                      icon={IconFromNotification(item)}
+                      style={{ backgroundColor: '#7B849B' }}
+                    />
+                  </div>
+                </div>
+                <div style={{ marginLeft: '10px', overflow: 'hidden' }}>
+                  <Typography.Paragraph
+                    style={{ fontSize: '14px' }}
+                    ellipsis={{ rows: 1, expandable: true }}
```

Comment:
May use `symbol` to input expand translation
```suggestion
                    ellipsis={{ rows: 1, expandable: true, symbol: t('general.notification.expand') }}
```